qa/vagrant/src/test/resources/packaging/tests/70_sysv_initd.bats

```diff
@@ -163,3 +163,31 @@ setup() {
     assert_file_exist /var/log/elasticsearch/gc.log.0.current
     stop_elasticsearch_service
 }
+
+# Ensures that if $MAX_MAP_COUNT is less than the set value on the OS
+# it will be updated
+@test "[INIT.D] sysctl is run when the value set is too small" {
+  # intentionally a ridiculously low number
+  sysctl -q -w vm.max_map_count=100
+  start_elasticsearch_service
+  max_map_count=$(sysctl -n vm.max_map_count)
+  stop_elasticsearch_service
+
+  [ $max_map_count != 100 ]
```

Comment:
I would be okay with us asserting that this is `262144`.

Reply:
I personally do not think it would be better as if the defaults change this is one more test to update and if its not `100` and the previous command did not error (which I assume would trigger a failed build) it should essentially be the same but if you feel otherwise I can certainly change it.

Reply:
Well, today we want to ensure that the value is set to what we specify: 262144. It's not much overhead to have to change this if we were to make a change to the default here.

Reply:
k, will update.

Reply:
updated